airflow/executors/base_executor.py

```diff
@@ -202,11 +216,15 @@ def get_event_buffer(self, dag_ids=None):
         return cleared_events
 
     def execute_async(self,
-                      key: ExecutorKeyType,
+                      key: TaskInstanceKey,
                       command_to_run: CommandType,
                       executor_config: Any = None):  # pragma: no cover
         """
         This method will execute the command asynchronously.
+
+        :param key: Unique key for the task instance
+        :param command_to_run: Command to run
```

Comment:
`:param command_to_run: Command to run` ?

Can we change this to just `command` ?

Reply:
Yep. It was more of an experiment :).